hal/src/nRF52840/gsm0710muxer/platform.h

```diff
@@ -45,6 +45,7 @@ const auto taskPriority = OS_THREAD_PRIORITY_NETWORK;
 // #define GSM0710_MODEM_STATUS_SEND_EMPTY_BREAK
 // #define GSM0710_PUMP_INPUT_DATA 50
 // #define GSM0710_ENABLE_DEBUG_LOGGING
+// FIXME:
```

Comment:
Fix what?

Reply:
:+1:
No longer relevant, should be removed.